imports/addCommand/server.lua

```diff
@@ -44,6 +45,15 @@ local function parseArguments(source, args, raw, params)
             if not value or not DoesPlayerExist(value--[[@as string]]) then
                 value = false
             end
+        elseif param.type == 'longString' and i == paramsNum then
+            value = arg
+
+            local argsNum = #args
+            if paramsNum < argsNum then
+                for a = i + 1, argsNum do
+                    value = value .. ' ' .. args[a]
+                end
+            end
```

Comment:
Rather than concatenating the rest of the passed arguments, just take the raw string and trim off the other arguments.

Reply:
Ah yeah I forgot we had a raw string